pgx-pg-sys/src/submodules/panic.rs

```diff
@@ -64,11 +65,18 @@ pub struct ErrorReportLocation {
     pub(crate) funcname: Option<String>,
     pub(crate) line: u32,
     pub(crate) col: u32,
+    pub(crate) backtrace: Option<Arc<std::backtrace::Backtrace>>,
```

Comment:
I think my only question, here, is if this *needs* to be `Arc`, as opposed to `Box`?

Reply:
Why can't it just be `Option<std::backtrace::Backtrace>`?

Reply:
Let me have a try...

Reply:
`ErrorReportLocation` requires `Clone`, so we have to use an Arc here.

Reply:
Oh, Backtrace doesn't derive Clone! That explains it. Thanks for taking a look at that!

Reply:
Just take `#[derive(Clone)]` off `ErrorReportLocation` and do the same with the resulting compilation errors?  We don't clone these things and I have no justification for why I did it in the first place.

Reply:
the diff looks like:

```diff
diff --git a/pgx-pg-sys/src/submodules/panic.rs b/pgx-pg-sys/src/submodules/panic.rs
index 57a8f968..472e51c5 100644
--- a/pgx-pg-sys/src/submodules/panic.rs
+++ b/pgx-pg-sys/src/submodules/panic.rs
@@ -58,7 +58,7 @@ where
     }
 }
 
-#[derive(Clone, Debug)]
+#[derive(Debug)]
 pub struct ErrorReportLocation {
     pub(crate) file: String,
     pub(crate) funcname: Option<String>,
@@ -106,7 +106,7 @@ impl From<&PanicInfo<'_>> for ErrorReportLocation {
 
 /// Represents the set of information necessary for pgx to promote a Rust `panic!()` to a Postgres
 /// `ERROR` (or any [`PgLogLevel`] level)
-#[derive(Clone, Debug)]
+#[derive(Debug)]
 pub struct ErrorReport {
     pub(crate) sqlerrcode: PgSqlErrorCode,
     pub(crate) message: String,
@@ -128,7 +128,7 @@ impl Display for ErrorReport {
     }
 }
 
-#[derive(Clone, Debug)]
+#[derive(Debug)]
 pub struct ErrorReportWithLevel {
     pub(crate) level: PgLogLevel,
     pub(crate) inner: ErrorReport,
@@ -294,7 +294,7 @@ impl CaughtError {
     }
 }
 
-#[derive(Debug, Clone)]
+#[derive(Debug)]
 enum GuardAction<R> {
     Return(R),
     ReThrow,
```

Reply:
sure, I'll fix it

Reply:
Thanks.  :)  `Arc` implies things about `ErrorReportLocation` that aren't actually true, so it'll be best to not need it.